frame/contracts/src/lib.rs

```diff
@@ -709,6 +734,33 @@ pub mod pallet {
 				T::WeightInfo::instantiate(data_len, salt_len),
 			)
 		}
+
+		/// When a migration is in progress, this dispatchable can be used to run migration steps.
+		/// Calls that contribute to advancing the migration have their fees waived, as it's helpful
+		/// for the chain. Note that while the migration is in progress, the pallet will also
+		/// leverage the `on_idle` hooks to run migration steps.
+		#[pallet::call_index(9)]
```

Comment:
note from talk with Sasha: should we give a different DispatchClass to this call?